### ISSUE TITLE
docker image failed

### ISSUE THREAD

**Issue by UrsZeidler** [pandoracore/bitcoin-pro#16]
Hi I tried the docker image but sadly it failed:

```
Step 8/11 : RUN . $HOME/.cargo/env && cargo install bitcoin-pro
 ---> Running in 8ef4e0826707
    Updating crates.io index
 Downloading crates ...
  Downloaded bitcoin-pro v0.1.0
  Installing bitcoin-pro v0.1.0
error: failed to compile `bitcoin-pro v0.1.0`, intermediate artifacts can be found at `/tmp/cargo-installg6pXWn`

Caused by:
  failed to select a version for the requirement `zeroize = "^0.9"`
  candidate versions found which didn't match: 1.3.0, 1.2.0, 1.1.1, ...
  location searched: crates.io index
  required by package `grin_secp256k1zkp v0.7.10`
      ... which is depended on by `rgb-core v0.4.0`
      ... which is depended on by `bitcoin-pro v0.1.0`

```
maybe I miss something ?

**Comment by dr-orlovsky** [comment]
Unfortunately this issue is unrelated to Bitcoin Pro and caused by `zeroise` bug which caused its creators to remove versions of the crate used by our upstream dependency (grin Secp256k1 library implementing bulletproofs). Waiting for them to fix this problem

**Comment by dr-orlovsky** [comment]
Ok, upstream is fixed and I did a new `v0.1.1` release, so pls re-check

**Comment by UrsZeidler** [comment]
works
